src/citrine/informatics/processors.py

```diff
@@ -25,9 +26,22 @@ def get_type(cls, data) -> Type['Processor']:
         """Return the sole currently implemented subtype."""
         return {
             'Grid': GridProcessor,
-            'Enumerated': EnumeratedProcessor
+            'Enumerated': EnumeratedProcessor,
+            'ContinuousSearch': MonteCarloProcessor
```

Comment:
Why call it `MonteCarlo` instead of `ContinuousSeach`?

Reply:
There are other ways we could do continuous search, and you can do monte carlo on discrete spaces too.